h/claim/views.py

```diff
@@ -28,7 +28,7 @@ def claim_account(request):
 
 @view_config(route_name='claim_account',
              request_method='POST',
-             renderer='h:claim/templates/claim_account.html.jinja2')
+             renderer='h:teampltes/claim/claim_account.html.jinja2')
```

Comment:
Wow. I have no idea how I managed that -- I didn't do these by hand. Anyway, thank you for the eagle eyes.